app/src/main/java/agency/five/codebase/android/movieapp/ui/component/FavoriteButton.kt

```diff
@@ -0,0 +1,44 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.R
+import androidx.compose.runtime.*
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.ui.Modifier
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun FavoriteButton (
+    modifier: Modifier=Modifier
+) {
+
+    var isFavorite by rememberSaveable { mutableStateOf(true) }
+
+    Image(
+        painter = painterResource(
+            id = if (isFavorite) R.drawable.favoritefull else R.drawable.favoriteempty
+        ),
+        contentDescription = null,
+
+        modifier = Modifier
```

Comment:
Since you have one composable (Image) in FavoriteButton(), you can use `modifier` parameter here instead of creating a new instance with `Modifier`